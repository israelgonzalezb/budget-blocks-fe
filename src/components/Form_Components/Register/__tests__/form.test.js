
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import form from '../form';


describe('form test', () => {
  it('form should match snapshot', () => {
    const component = renderer.create(<form
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
