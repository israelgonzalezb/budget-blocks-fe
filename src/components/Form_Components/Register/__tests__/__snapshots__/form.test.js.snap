// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form test form should match snapshot 1`] = `<form />`;
